src/containers/skillProgress/Progress.css

```diff
@@ -1,69 +1,75 @@
-.meter { 
-    height: 20px;
-    position: relative;
-    background: rgb(243, 239, 239);
-    -moz-border-radius: 25px;
-    -webkit-border-radius: 25px;
-    border-radius: 25px;
+.meter {
+  height: 20px;
+  position: relative;
+  background: rgb(243, 239, 239);
+  -moz-border-radius: 25px;
+  -webkit-border-radius: 25px;
+  border-radius: 25px;
 }
+
 .meter > span {
-    display: block;
-    height: 100%;
-    border-top-right-radius: 6px;
-    border-bottom-right-radius: 6px;
-    border-top-left-radius: 20px;
-    border-bottom-left-radius: 20px;
-    background-color: #aaa5ff;
-    position: relative;
-    overflow: hidden;
+  display: block;
+  height: 100%;
+  border-top-right-radius: 6px;
+  border-bottom-right-radius: 6px;
+  border-top-left-radius: 20px;
+  border-bottom-left-radius: 20px;
+  background-color: #aaa5ff;
+  position: relative;
+  overflow: hidden;
 }
-.skill{
-    line-height: 3.5vw;
+
+.skill {
+  line-height: 3.5vw;
 }
-.skill-bars{
-    font-size: 28px;
-    width: 40%;
+
+.skill-bars {
+  font-size: 28px;
+  width: 40%;
 }
+
 .skills-container {
-    display: flex;
-    width: 90%;
-    padding: 20px 10px;
-    margin: 0px auto;
-    margin-top: 2rem;
+  display: flex;
+  width: 90%;
+  padding: 20px 10px;
+  margin: 0 auto;
+  margin-top: 2rem;
 }
+
 .skills-container > * {
-    flex: 1;
-    margin-bottom: 30px;
+  flex: 1;
+  margin-bottom: 30px;
 }
+
 .skills-image > img {
-    margin-left: 80px;
-    max-width: 90%;
-    height: auto;
+  margin-left: 80px;
+  max-width: 90%;
+  height: auto;
 }
 
 /* Media Query */
-
 @media (max-width: 1456px) {
-    .skills-bar {
-        line-height: 4rem;
-    }
-    .skills-image {
-        display: none;
-        order: 2;
-    }
```

Comment:
This wasn't a bug, wait for me to confirm on the bug before starting on it. It works completely fine and how its supposed to. Your screen resolution must be 1366 or similar that is why you don't see it on your deploy.

Remove this file from commit too.

Reply:
OK.

Reply:
I have done the formating of this file. I would keep that. But I will add the line `display: none;` to skills-image.

Reply:
It wouldn't matter since #107 will require re-formatting all the files so let's keep it for that PR.

Reply:
Uncommit Progress.css for your PR since formatting will have to be done again anyways and this isn't related to this PR